libs/components/src/tabs/vl-tabs.component.ts

```diff
@@ -268,15 +272,15 @@ export class VlTabsComponent extends BaseHTMLElement {
         this._addTab({ tabPane, index });
 
         if (!this.hasAttribute('within-functional-header')) {
-            this._addTabSection({ id: tabPane.paneId, index });
+            this._addTabSection({ id: tabPane.getAttribute('data-vl-id') || '', index });
```

Comment:
ik denk dat je wil: als tabPane.getAttribute('data-vl-id') == null dan moet je '' gebruiken
zo ja, dan kan je beter ?? gebruiken, is niet helemaal hetzelfde als ||
(maar geeft in dit geval waarschijnlijk wel hetzelfde resultaat)
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or